expyfun/_eyelink_controller.py

```diff
@@ -137,14 +137,17 @@ class EyelinkController(object):
         Sample rate to use. Must be one of [250, 500, 1000, 2000].
     verbose : bool, str, int, or None
         If not None, override default verbose level (see expyfun.verbose).
+    calbration_keys : list
+        Keys that will trigger recalibration when check_recalibration.
```

Comment:
can you expand this docstring line a bit?  The reference to `check_calibration` needs context (i.e., it's not a parameter of this constructor, so explain what it is / where it lives)...  is it referring to the new `check_recalibrate` method?  (if so, the method name is wrong here)

Reply:
also, you say it should be `list` but you initiate it with a tuple.  Will it work with any iterable?  if so, say `iterable` instead of `list`